### PR TITLE
Icon scaling fixes

### LINKED ISSUE #541: Icon: Fix 20 -> 24 scaling, and add arrow-left-small for Beacon adapter
The `transform: scale()` math needs to be `1.1` instead of `1.2`.

Even though `1.2` is correct (mathematically), in practice, it's a little too large.

Screen Shot 2019-03-19 at 7 12 39 PM

Above is a screenshot showing the overlay of a `1.1` scaled icon on top of the desired Zeplin icon mock.

### PR DESCRIPTION
**Pull request by ItsJonQ**
## Icon scaling fixes

This update fixes the icon scaling issue, which resulted from recent Icon SVG changes to HSDS: React.

This update removes the scaling of `24x24` based icons. The prior solution resulted in unexpectedly large icons when rendered within the App + Embed.

Instead, the scaling is applied to the older icons (20x20), to ensure that they match the final rendered size of the newer set.

## Screencast

https://www.useloom.com/share/3a4cdaf137dc45deb318ceffb8eac3ee

## Deploy preview

https://deploy-preview-543--hsds-react.netlify.com/?selectedKind=Icon&selectedStory=icons&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

cc'ing @jaredmcdaniel 

Resolves #541

### CODE REVIEW

**Comment by ItsJonQ** [comment]
## Deploy preview

https://deploy-preview-543--hsds-react.netlify.com/?selectedKind=Icon&selectedStory=icons&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

cc'ing @jaredmcdaniel

File: stories/Icon.stories.js [hunk 1]
```diff
@@ -41,10 +42,28 @@ const IconWrapper = styled('div')`
...
+      18: 18,
+      20: 20,
+      24: 24,
+      32: 32,
+      48: 48,
+      52: 52,
+    },
+    20
+  )
+
```
Comment (plbabin) [inline stories/Icon.stories.js]: nice addition 💯

**Comment by ItsJonQ** [comment]
Pausing this for now. We'll have a clearer idea of what we need to do after a chat with @jaredmcdaniel and @digitaldesigner .

For now, we have HOTFIX

https://github.com/helpscout/hsds-react/pull/544

**Comment by ItsJonQ** [comment]
Closing, in favour of https://github.com/helpscout/hsds-react/pull/547

**Pull request closed without merging**